core/templates/pages/skill-editor-page/navbar/skill-editor-navbar.directive.ts

```diff
@@ -128,9 +128,16 @@ angular.module('oppia').directive('skillEditorNavbar', [
             // discarded, the misconceptions won't be saved, but there will be
             // some questions with these now non-existent misconceptions.
             if (UndoRedoService.getChangeCount() > 0) {
-              NgbModal.open(SavePendingChangesModalComponent, {
-                backdrop: true,
-              }).result.then(null, function() {
+              const modalRef = NgbModal.open(
+                SavePendingChangesModalComponent, {
+                  backdrop: true
+                });
+
+              modalRef.componentInstance.body =
+                'Please save all pending ' +
+                'changes before viewing the questions list.';
```

Comment:
Here and elsewhere, use extra parenthesis for statements that break into multiple lines. (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)).

Reply:
Done